azure-pipelines.yml

```diff
@@ -51,25 +49,32 @@ jobs:
         displayName: 'Install Dependencies'
 
       - script: |
-          go vet ./autorest/...
-          go vet ./logger/...
-          go vet ./tracing/...
+          modules=$(find . -name go.mod)
+          for module in $modules; do
+            pushd "$(dirname $module)" && go vet ./... && popd;
+          done
```

Comment:
Each module is now it's own "thing", so looks like we need to iterate over all modules and run vet, lint, test separately. `gofmt` seems to be the exception (it didn't like iterating 🤔)